benchmark/Streamly/Benchmark/Data/NestedUnfoldOps.hs

```diff
@@ -5,7 +5,7 @@
 -- License     : BSD3
 -- Maintainer  : streamly@composewell.com
 
-module NestedUnfoldOps where
+module Streamly.Benchmark.Data.NestedUnfoldOps where
```

Comment:
1. Rename this to `Data.Unfold` to reflect the path of the module in the library.
2. Move the unfold benchmarks from Prelude here.

Reply:
Yes, I've done it already, didn't push it yet.